### PR TITLE
Sc 1945

### PR DESCRIPTION
**Pull request by rheafernandes**

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ asset_prefix }}/v1/update"
+    upstream_url: "{{ content_service_url }}/asset/v4/update"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5292]: ACL need to be contentUpdate @rheafernandes

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ asset_prefix }}/v1/read"
+    upstream_url: "{{ content_service_url }}/asset/v4/read"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5310]: ACL should be contentAccess

File: ansible/roles/kong-api/defaults/main.yml [hunk 3]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: uploadAsset
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5318]: where will this API be used?

File: ansible/roles/kong-api/defaults/main.yml [hunk 4]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: uploadUrlAsset
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5336]: need more information on this API

File: ansible/roles/kong-api/defaults/main.yml [hunk 5]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ asset_prefix }}/v1/retire"
+    upstream_url: "{{ content_service_url }}/asset/v4/retire"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5364]: ACL value has to be contentAdmin

File: ansible/roles/kong-api/defaults/main.yml [hunk 6]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ collection_prefix }}/v1/read"
+    upstream_url: "{{ content_service_url }}/collection/v4/read"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5400]: ACL should be contentAccess

File: ansible/roles/kong-api/defaults/main.yml [hunk 7]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ collection_prefix }}/v1/update"
+    upstream_url: "{{ content_service_url }}/collection/v4/update"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5418]: ACL has to be contentUpdate

File: ansible/roles/kong-api/defaults/main.yml [hunk 8]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: collectionFlag
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5426]: need more information on this API!

File: ansible/roles/kong-api/defaults/main.yml [hunk 9]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: collectionDiscard
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5444]: Need more information on this API

File: ansible/roles/kong-api/defaults/main.yml [hunk 10]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ collection_prefix }}/v1/retire"
+    upstream_url: "{{ content_service_url }}/collection/v4/retire"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5472]: ACL has to be contentAdmin

File: ansible/roles/kong-api/defaults/main.yml [hunk 11]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ collection_prefix }}/v1/copy"
+    upstream_url: "{{ content_service_url }}/collection/v4/copy"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5581]: ACL has to be contentUpdate/contentAdmin based on the functionmality, need more information on this @rheafernandes

File: ansible/roles/kong-api/defaults/main.yml [hunk 12]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+
+  - name: collectionDialcodeLink
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5553]: need more information on this!

File: ansible/roles/kong-api/defaults/main.yml [hunk 13]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: collectionHierarchyRemove
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5498]: Need more information on this!

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (rheafernandes) [reply ansible/roles/kong-api/defaults/main.yml]: done

File: ansible/roles/kong-api/defaults/main.yml [hunk 14]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ collection_prefix }}/v1/read"
+    upstream_url: "{{ content_service_url }}/collection/v4/read"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentUpdate'
```
Comment (amitpriyadarshi) [inline ansible/roles/kong-api/defaults/main.yml:5400]: use contentAccess

File: ansible/roles/kong-api/defaults/main.yml [hunk 15]
```diff
@@ -5257,3 +5260,328 @@ kong_apis:
...
+    uris: "{{ collection_prefix }}/v1/dialcode/link"
+    upstream_url: "{{ content_service_url }}/collection/v4/dialcode/link"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+       - 'contentCreate'
```
Comment (amitpriyadarshi) [inline ansible/roles/kong-api/defaults/main.yml:5563]: use contentUpdate

**Review by amitpriyadarshi — APPROVED** [review]

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**